sound/soc/sof/intel/hda.c

```diff
@@ -607,18 +607,19 @@ int hda_dsp_probe(struct snd_sof_dev *sdev)
 	 * TODO: support interrupt mode selection with kernel parameter
 	 *       support msi multiple vectors
 	 */
-//	ret = pci_alloc_irq_vectors(pci, 1, 1, PCI_IRQ_MSI);
+	ret = pci_alloc_irq_vectors(pci, 1, 1, PCI_IRQ_MSI);
```

Comment:
maybe fix the TODO below? It's my understanding that we sometimes miss an HDMI interrupt, but that happens for both MSI and ioapic.

Reply:
Yes, I should remove the below TODO. Thanks for point it out. We sometimes miss HDMI interrupts in both MSI mode and IO-APIC mode. When there is no interrupts, the playback will be pending. I believe this is another bug. Maybe this is HW related as my UP2 is easier to catch this issue than CNL. Actually I tested about 20 times on CNL and didn't find this issue.